src/analysis/ptyp_of_type.ml

```diff
@@ -34,8 +34,12 @@ let rec module_type =
 and core_type type_expr =
   let open Ast_helper in
   match Types.get_desc type_expr with
-  | Tvar { name = None; _ } | Tunivar { name = None; _ } -> Typ.any ()
-  | Tvar { name = Some s; _ } | Tunivar { name = Some s; _ } -> Typ.var s
+  | Tvar { name = None; jkind = _ } | Tunivar { name = None; jkind = _ } ->
+    (* CR modes: do something better here with the jkind *)
```

Comment:
Seems easy enough to do better -- but maybe the point is that we're not sure what we want to display to users? (This doesn't need to hold up the merge.)

Reply:
I'm not sure what we want to display to the user, and I'd rather print the jkind to little than too often (this is only used for construct), so I think it's best to just not include the kind for now. I also suspect this might be easier to do once https://github.com/ocaml-flambda/flambda-backend/pull/3224 is merged.